### PR TITLE
NathanBeals.WinSSH-Pageant version 2.0.0

### PR DESCRIPTION
**Pull request by ndbeals**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/54828)

### CODE REVIEW

File: manifests/n/NathanBeals/WinSSH-Pageant/2.0.0/NathanBeals.WinSSH-Pageant.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  InstallerType: wix
+  InstallerUrl: https://github.com/ndbeals/winssh-pageant/releases/download/v2.0/winssh-pageant-v2.0_amd64.msi
+  InstallerSha256: 9B0C885ABCBE0F39251135EB330A411990F95CFA29A04787A540FAAF30D60F79
+  ProductCode: '{839A2310-DE92-47F9-B5DB-8F1E41ABA405}'
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://github.com/ndbeals/winssh-pageant/releases/download/v2.0/winssh-pageant-v2.0_386.msi
+  InstallerSha256: 040467392C2857661C0662A185F8259B48A842C7DD7E322E1806EC1376119E9F
+  ProductCode: '{B5A49790-199C-4BA6-8E4A-B28CA56A04F8}'
```
Comment (ItzLevvie) [inline manifests/n/NathanBeals/WinSSH-Pageant/2.0.0/NathanBeals.WinSSH-Pageant.installer.yaml:18]:
```suggestion
  ProductCode: '{839A2310-DE92-47F9-B5DB-8F1E41ABA405}'
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Comment by ndbeals** [comment]
Okay, so do I just wait for the wingetbot to comment the "/AzurePipelines run" command?

**Comment by ndbeals** [comment]
I had to make some code changes to the application (winssh-pageant) which means there's a version bump in that application. Closing this PR to open a new one with the new version.

**Comment by ndbeals** [comment]
I had to make some code changes to the application (winssh-pageant) which means there's a version bump in that application. Closing this PR to open a new one with the new version. Apologies for the spam in this PR. New one is here: https://github.com/microsoft/winget-pkgs/pull/55240

**Pull request closed without merging**